.github/workflows/cd.yml

```diff
@@ -36,6 +44,13 @@ jobs:
     steps:
       - name: Git Checkout
         uses: actions/checkout@v2
+      - name: serializing workflow runs
```

Comment:
I don't think the deployment job requires the turnstyle step. As it only runs when benchmark_tests succeed.

Reply:
88e411c11